site/content/en/docs/v0.1/quickstart.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Quickstart
+weight: 0
+description: Get etcd with etcd-operator up and running in less than 5 minutes!
+---
+
+Follow these instructions to install, run, and test etcd with etcd-operator in a Kubernetes cluster.
+
+Pre-requisites:
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [kustomize](https://github.com/kubernetes-sigs/kustomize)
+- Kubernetes cluster and `kubectl` configured to use it
+  - If you don't have a Kubernetes cluster, you can use [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) to create a local one
+- [cert-manager](https://cert-manager.io/docs/installation/) installed in the cluster
+
+1. Install etcd-operator:
+    ```bash
+    kustomize build 'https://github.com/aenix-io/etcd-operator//config/default?ref=main' | kubectl apply -f -
```

Comment:
Just an idea:

```suggestion
    VERSION=$(curl -sS https://api.github.com/repos/aenix-io/etcd-operator/releases/latest | awk -F '"' '/tag_name/ {print $4}')
    kustomize build "https://github.com/aenix-io/etcd-operator//config/default?ref=$VERSION" | kubectl apply -f -
```

Reply:
Yeah, sorry this won't work because ref will just take yamls from git, and there is :latest